nornir/plugins/connections/netconf.py

```diff
@@ -108,9 +109,10 @@ def open(
 
         if "ssh_config" not in extras:
             try:
-                parameters["ssh_config"] = configuration.ssh.config_file  # type: ignore
+                ssh_config_file = Path(configuration.ssh.config_file)  # type: ignore
+                if ssh_config_file.exists():
+                    parameters["ssh_config"] = ssh_config_file
```

Comment:
I think this can perhaps become a bit confusing if a user has made a typo in the name of the config file, that way the supplied file wouldn't exist and the user wouldn't be notified about that problem. But I'm also fine with this small fix for Nornir 2.0 and then I can change it for 3.0.